morphir/runtime/src/org/finos/morphir/runtime/TestSummary.scala

```diff
@@ -25,7 +45,8 @@ case class TestSummary(
   override def toString =
     s"$report\n" +
       s"$overallCounts\n" +
-      s"Overal Status - $result"
+      s"Overall Status - $result\n" +
+      s"Coverage by package - $overallCoverage"
```

Comment:
We should improve the pretty-printing of this - format it with something more than the default Map toString